packages/replayio/src/commands/upload-sourcemaps.ts

```diff
@@ -0,0 +1,63 @@
+import { uploadSourceMaps as uploadSourceMapsExternal } from "@replayio/sourcemap-upload";
+import { replayApiServer } from "../config";
+import { getAccessToken } from "../utils/authentication/getAccessToken";
+import { registerCommand } from "../utils/commander/registerCommand";
+import { exitProcess } from "../utils/exitProcess";
+import { dim } from "../utils/theme";
+import { logPromise } from "../utils/async/logPromise";
+
+registerCommand("upload-sourcemaps <paths...>", { requireAuthentication: true })
```

Comment:
I removed a couple of options from here because they felt redundant. For instance, there is no need for `--api-key` when we support `REPLAY_API_KEY` already. `batch-size` felt like something the user shouldn't care about - so why do we bother exposing it? `dry-run`, `verbose` and `quiet` could be useful but I think it's a bigger question and if anything them (or some of them) should be introduced to the whole new CLI - having them just here would be weird.

Reply:
Agreed with those simplifications.